ush/wave_ens_bull.sh

```diff
@@ -0,0 +1,260 @@
+#!/bin/bash
+#                                                                       
+################################################################################
+#
+# UNIX Script Documentation Block
+# Script name:         wave_ens_bull.sh
+# Script description:  Create buoy bulletin for NCEP Global Wave Ensemble
+#
+# Author:   Jose-Henrique Alves Org: NCEP/EMC      Date: 2014-01-16
+# Abstract: Creates bulletin for NCEP Global Wave Ensemble using grib2 data.
+#           Values at buoy locations are extracted using wgrib2 bi-linear 
+#           interpolation (-new_grid) and requires IPOLATES lib.
+#
+# Script history log:
+# 2019-05-06  J-Henrique Alves First Version.
+# 2019-11-02  J-Henrique Alves Ported to global-workflow.
+#
+# $Id$
+#
+# Attributes:
```

Comment:
This could probably be removed since we're no longer on dell-p3

Reply:
@JessicaMeixner-NOAA will do.